data/sql_updates/create_totals_pacs_parties_view.sql

```diff
@@ -0,0 +1,49 @@
+drop materialized view if exists ofec_totals_pacs_parties_mv;
+create materialized view ofec_totals_pacs_parties_mv as
+select
+    cmte_id as committee_id,
+    two_yr_period_sk as cycle,
+    cmte_tp as committee_type,
+    sum(all_loans_received_per) as all_loans_received,
+    sum(ttl_contb_ref_per_i) as contribution_refunds,
+    sum(ttl_contb_per) as contributions,
+    sum(coord_exp_by_pty_cmte_per) as coordinated_expenditures_by_party_committee,
+    sum(ttl_disb_per) as disbursements,
+    sum(fed_cand_cmte_contb_per) as fed_candidate_committee_contributions,
+    sum(fed_cand_contb_ref_per) as fed_candidate_contribution_refunds,
+    sum(ttl_fed_disb_per) as fed_disbursements,
+    sum(ttl_fed_elect_actvy_per) as fed_elect_activity,
+    sum(ttl_fed_op_exp_per) as fed_operating_expenditures,
+    sum(ttl_fed_receipts_per) as fed_receipts,
+    sum(indt_exp_per) as independent_expenditures,
+    sum(indv_contb_ref_per) as individual_contribution_refunds,
+    sum(indv_item_contb_per) as individual_itemized_contributions,
+    sum(indv_unitem_contb_per) as individual_unitemized_contributions,
+    sum(loan_repymts_made_per) as loan_repayments_made,
+    sum(loan_repymts_received_per) as loan_repayments_received,
+    sum(loans_made_per) as loans_made,
+    sum(net_contb_per) as net_contributions,
+    sum(non_alloc_fed_elect_actvy_per) as non_allocated_fed_election_activity,
+    sum(ttl_nonfed_tranf_per) as nonfed_transfers,
+    sum(offsets_to_op_exp_per_ii) as offsets_to_operating_expenditures,
+    sum(ttl_op_exp_per) as operating_expenditures,
+    sum(other_disb_per) as other_disbursements,
+    sum(other_fed_op_exp_per) as other_fed_operating_expenditures,
+    sum(other_fed_receipts_per) as other_fed_receipts,
+    sum(other_pol_cmte_contb_per_i) as other_political_committee_contributions,
+    sum(pol_pty_cmte_contb_per_i) as political_party_committee_contributions,
```

Comment:
I don't see pol_pty_cmte_contb_per_ii in the totals PAC and parties, those are `political_party_committee_contribution_refunds`